sdk/storage/mgmt/src/main/java/com/azure/management/storage/StorageManagementClientBuilder.java

```diff
@@ -96,11 +85,11 @@ public StorageManagementClientBuilder pipeline(HttpPipeline pipeline) {
     }
 
     /**
-     * Builds an instance of StorageManagementClientImpl with the provided parameters.
+     * Builds an instance of StorageManagementClient with the provided parameters.
      *
-     * @return an instance of StorageManagementClientImpl.
+     * @return an instance of StorageManagementClient.
      */
-    public StorageManagementClientImpl buildClient() {
+    public StorageManagementClient buildClient() {
         if (host == null) {
             this.host = "https://management.azure.com";
         }
```

Comment:
Do you still need the null check for `apiVersion` now that the setter is removed?

Reply:
I will try to address it when changing generator.